store/src/main/java/org/apache/rocketmq/store/index/IndexFile.java

```diff
@@ -71,6 +72,9 @@ public void load() {
         this.indexHeader.load();
     }
 
+    /**
+     * 头部文件刷盘 时间戳 msgId 物理偏移量..
```

Comment:
Why do you use so many annotations, and it is in Chinese?

Reply:
sory, I don't know why this code was submitted here

Reply:
this code just my study comment. i do'not want to pr the "3eee57d"

Reply:
this code just my study comment. i don't want to pr "3eee57d"